src/utilities/internationalization.js

```diff
@@ -0,0 +1 @@
+export const intDate = (date) => new Intl.DateTimeFormat('ru').format(new Date(date));
```

Comment:
why is it named so strange ? give it a clear name please

Reply:
It means internationalizedDate:
```suggestion
export const internationalizedDate = (date) => new Intl.DateTimeFormat('ru').format(new Date(date));
```

Reply:
For me it means `integer date`. Be careful with naming it can be very confusing.